ui/docs/style-guide.md

```diff
@@ -1,4 +1,71 @@
-# UI Style Guide
+# UI Coding Style Guide
+
+## Starting new work
+
+When starting a UI card, always check Invision for latest designs.
+	https://projects.invisionapp.com/d/main#/projects/prototypes/12820961
+	
+We are working towards abstracting styles (and possibly components) where it makes sense. When starting a new UI ticket, look around. 
+* Are there other places where elements similar to ones you are using are used? 
+* If so, have we already abstracted out the style and/or component?
+* If not, should you abstract out the style/component as part of your card?
+
+
+## React Component Structure
+
+  - Declare Props and State as interfaces
```

Comment:
I removed the 1.1 and 1.2 from these, because honestly that seems like a maintenance burden to have numbers strictly defined. I could see the top level headers having numbers, but markdown probably could do that automatically for us, so that when we move sections around they automatically change.